bin/testRelease.sh

```diff
@@ -2,17 +2,19 @@
 set -eux
 
 VERSION=$1
-coursier fetch com.lihaoyi:utest_2.10:$VERSION
-coursier fetch com.lihaoyi:utest_2.11:$VERSION
-coursier fetch com.lihaoyi:utest_2.12:$VERSION
-coursier fetch com.lihaoyi:utest_2.13.0-M5:$VERSION
-coursier fetch com.lihaoyi:utest_2.13.0-M5:$VERSION
-coursier fetch com.lihaoyi:utest_native0.3_2.11:$VERSION
-coursier fetch com.lihaoyi:utest_sjs0.6_2.10:$VERSION
-coursier fetch com.lihaoyi:utest_sjs0.6_2.11:$VERSION
-coursier fetch com.lihaoyi:utest_sjs0.6_2.12:$VERSION
-coursier fetch com.lihaoyi:utest_sjs0.6_2.13.0-M5:$VERSION
-coursier fetch com.lihaoyi:utest_sjs0.6_2.13.0-M5:$VERSION
-coursier fetch com.lihaoyi:utest_sjs1.0.0-M5_2.11:$VERSION
-coursier fetch com.lihaoyi:utest_sjs1.0.0-M5_2.12:$VERSION
-coursier fetch com.lihaoyi:utest_sjs1.0.0-M5_2.13.0-M5:$VERSION
+coursier fetch \
+  com.lihaoyi:utest_2.10:$VERSION \
+  com.lihaoyi:utest_2.11:$VERSION \
+  com.lihaoyi:utest_2.12:$VERSION \
+  com.lihaoyi:utest_2.13.0-M5:$VERSION \
+  com.lihaoyi:utest_2.13.0-M5:$VERSION \
+  com.lihaoyi:utest_native0.3_2.11:$VERSION \
+  com.lihaoyi:utest_sjs0.6_2.10:$VERSION \
+  com.lihaoyi:utest_sjs0.6_2.11:$VERSION \
+  com.lihaoyi:utest_sjs0.6_2.12:$VERSION \
+  com.lihaoyi:utest_sjs0.6_2.13.0-M5:$VERSION \
+  com.lihaoyi:utest_sjs0.6_2.13.0-M5:$VERSION \
+  com.lihaoyi:utest_sjs1.0.0-M5_2.11:$VERSION \
+  com.lihaoyi:utest_sjs1.0.0-M5_2.12:$VERSION \
+  -r sonatype:releases
+  # com.lihaoyi:utest_sjs1.0.0-M5_2.13.0-M5:$VERSION \ # Did not release
```

Comment:
@sjrd is this resolution error expected?

```
[error] sbt.librarymanagement.ResolveException: unresolved dependency: org.scala-js#scalajs-library_2.13.0-M5;1.0.0-M5: not found
```

Happened during release https://travis-ci.org/lihaoyi/utest/jobs/433990118#L2762

Reply:
Yes. Scala.js 1.0.0-M5 does not support Scala 2.13.0-M5.

Reply:
OK, let me disable it in .travis.yml then.